src/main/g8/pom.xml

```
@@ -12,17 +12,36 @@
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
     <akka.version>$akka_version$</akka.version>
-    <akka.grpc.version>$akka_grpc_version$</akka.grpc.version>
+    <akka.grpc.version>1.0.0-M1+35-c8aa943b</akka.grpc.version>
```

in gradle it's `1.0.0-M1+37-732f2dee`
```suggestion
    <akka.grpc.version>1.0.0-M1+37-732f2dee</akka.grpc.version>
```